src/Controller/JourneyController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Controller;
+
+use App\Domain\Journey\JourneyStepHelper;
+use App\Entity\Journey;
+use App\Entity\JourneyStep;
+use App\Form\JourneyType;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * Class JourneyController
+ */
+class JourneyController extends Controller
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * JourneyController constructor.
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * @param Request $request
+     * @param EntityManager $entityManager
+     * @return \Symfony\Component\HttpFoundation\Response
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     *
+     * @Route("/journey/create", name="journey_create")
```

Comment:
You can prefix route for all controller `@Route("/journey")` in class annotation. 
It's better to prevent duplication code and improve uniformity of path

Reply:
👍